server/index.js

```diff
@@ -19,6 +19,51 @@ const jsonMiddleWare = express.json();
 app.use(staticMiddleware);
 app.use(jsonMiddleWare);
 
+app.get('/api/accounts/:accountId', (req, res, next) => {
+  const accountId = req.params.accountId;
+  if (!accountId) {
+    throw new ClientError(400, 'username must be provided');
+  }
+
+  const total = `select count(*) as "reviews"
+  from "reviews"
+  where "accountId" = $1
+  group by "accountId"`;
+
+  const sql = `
+    select "stateCode",
+    count(*) as "visits"
+    from "accounts"
+    join "reviews" using ("accountId")
+    join "parksCache" using ("parkCode")
+    where "accountId" = $1
+    group by "stateCode"
+    order by "visits" desc`;
+
+  const params = [accountId];
+  db.query(sql, params)
+    .then(result => {
+      const visits = result.rows;
+      if (!visits) {
+        res.status(404).json({
+          error: 'Cannot find reviews/states for account'
+        });
+      }
```

Comment:
A couple of points here:
- arrays are always truthy, even if they're empty. a `pg` `Result` object always has a `rows` property, it just may be empty.
- typically an empty list is fine, we just send it back empty.